src/Dropdown/index.js

```diff
@@ -0,0 +1,110 @@
+import React, { Children, PureComponent } from 'react';
+import PropTypes from 'prop-types';
+import posed, { PoseGroup } from 'react-pose';
+import styled from 'styled-components';
+import onClickOutside from 'react-onclickoutside';
+
+import { Header, Menu, MenuItem } from './elements';
+
+const { node, string } = PropTypes;
+
+/**
+ * Dropdown
+ *
+ * This component is in charge of displaying
+ * a dropdown
+ *
+ * @param {string} children // Anything that can be rendered: numbers, strings, elements or an array (or fragment).
+ * @param {string} className // Add a text aside in the select next the selected value.
+ * @param {string} title // Dropdown title.
+ *
+ * @return {jsx}
+ */
+
+class Dropdown extends PureComponent {
+  /** Prop types. */
+  static propTypes = {
+    children: node.isRequired,
+    className: string,
+    title: node.isRequired,
+  };
+
+  /** Default props. */
+  static defaultProps = {
+    className: '',
+  };
+
+  /** Internal state. */
+  state = {
+    displayMenu: false,
+  };
+
+  /**
+   * Toogle Menu
+   */
+  toggleMenu = () => {
+    this.setState(prevState => ({
+      displayMenu: !prevState.displayMenu,
+    }));
+  };
+
+  /**
+   * Handle Click Outside
+   */
+  handleClickOutside = () => {
+    this.setState({ displayMenu: false });
+  };
+
+  render() {
+    const { children, className, title } = this.props;
+    const { displayMenu } = this.state;
+
+    return (
+      <div className={className}>
+        <Header
+          onClick={this.toggleMenu}
+          aria-haspopup="true"
+          aria-expanded={displayMenu}
+          data-test="dropdown-header"
+        >
+          {title}
```

Comment:
Here title should be dynamic depending on selected value, isn't it ?

Reply:
@AntoninMarchardDev  Ok but you could have multiple selected value

Reply:
@spartDev as Jira does, each item selected are displayed in the title of the dropdown : 

<img width="262" alt="capture d ecran 2018-11-08 a 2 21 42 pm" src="https://user-images.githubusercontent.com/29677243/48201154-aa1a8e00-e361-11e8-869b-c1a6c96197ad.png">